scripts/test_priority_plugin.sh

```diff
@@ -66,6 +73,51 @@ function start_container() {
 	done
 }
 
+#!/bin/bash
+
+function compile_priority_plugin() {
+    if [ "$RELEASE_MODE" = true ]; then
+        cargo build --bin auditor-priority-plugin --target x86_64-unknown-linux-musl --release
+    else
+        cargo build --bin auditor-priority-plugin --target x86_64-unknown-linux-musl
+    fi
+}
+
+function start_priority_plugin() {
+    if [[ -z "${SKIP_COMPILATION}" ]]; then
+        compile_priority_plugin
+    fi
+
+    if [ "$RELEASE_MODE" = true ]; then
+        local config_file="$1"
+        docker exec "${COMPOSE_PROJECT_NAME}-slurm-1" /auditor-priority-plugin /$config_file &
+    else
+        local config_file="$1"
+        docker exec "${COMPOSE_PROJECT_NAME}-slurm-1" /auditor-priority-plugin /$config_file &
+    fi
+
+    COUNTER=0
+
+    until curl http://0.0.0.0:9000/metrics; do
+        >&2 echo "Priority plugin is still unavailable - sleeping"
+        ((COUNTER=COUNTER+1))
+
+        if [ "$COUNTER" -gt 30 ]; then
+            echo >&2 "Priority plugin did not come up in time"
+            stop_priority_plugin $PRIORITY_PLUGIN
```

Comment:
We don't need `$PRIORITY_PLUGIN` here, because this is not used in the `stop_priority_plugin` function